completions/rsync

```diff
@@ -36,32 +36,10 @@ _rsync()
 
     case $cur in
         -*)
-            COMPREPLY=($(compgen -W '--verbose --quiet --no-motd --checksum
-                --archive --recursive --relative --no-implied-dirs
-                --backup --backup-dir= --suffix= --update --inplace --append
-                --append-verify --dirs --old-dirs --links --copy-links
-                --copy-unsafe-links --safe-links --copy-dirlinks
-                --keep-dirlinks --hard-links --perms --executability --chmod=
-                --acls --xattrs --owner --group --devices --copy-devices
-                --specials --times --omit-dir-times --super --fake-super
-                --sparse --dry-run --whole-file --no-whole-file
-                --one-file-system --block-size= --rsh= --rsync-path=
-                --existing --ignore-existing --remove-source-files --delete
-                --delete-before --delete-during --delete-delay --delete-after
-                --delete-excluded --ignore-errors --force --max-delete=
-                --max-size= --min-size= --partial --partial-dir=
-                --delay-updates --prune-empty-dirs --numeric-ids --timeout=
-                --contimeout= --ignore-times --size-only --modify-window=
-                --temp-dir= --fuzzy --compare-dest= --copy-dest= --link-dest=
-                --compress --compress-level= --skip-compress= --cvs-exclude
-                --filter= --exclude= --exclude-from= --include= --include-from=
-                --files-from= --from0 --protect-args --address= --port=
-                --sockopts= --blocking-io --no-blocking-io --stats
-                --8-bit-output --human-readable --progress --itemize-changes
-                --out-format= --log-file= --log-file-format= --password-file=
-                --list-only --bwlimit= --write-batch= --only-write-batch=
-                --read-batch= --protocol= --iconv= --ipv4 --ipv6 --version
-                --help --daemon --config= --no-detach' -- "$cur"))
+            COMPREPLY=($(compgen -W '$(_parse_help "$1")
+                --config --daemon --old-d{,irs}
```

Comment:
Thank you for updating the PR! Could you drop `--config` from here, or do you think we can keep `--config` in this PR?

Reply:
Meh my bad, I removed it, then I copied some old code again. Let me fix that.

Reply:
No problem, thanks.